examples/jsm/renderers/webgpu/WebGPUBufferUtils.js

```diff
@@ -0,0 +1,15 @@
+import { GPUChunkSize } from './constants.js';
+
+class WebGPUBufferUtils {
+
+	static getFloatLength( floatLength ) {
+
+		// ensure chunk size alignment (STD140 layout)
+
+		return floatLength + ( ( GPUChunkSize - ( floatLength % GPUChunkSize ) ) % GPUChunkSize );
+
+	}
+
+}
+
+export default WebGPUBufferUtils;
```

Comment:
Let's use this style similar to other utils modules which is best for tree-shaking:
```js
import { GPUChunkSize } from './constants.js';

function getFloatLength( floatLength ) {

    // ensure chunk size alignment (STD140 layout)

    return floatLength + ( ( GPUChunkSize - ( floatLength % GPUChunkSize ) ) % GPUChunkSize );

}

export {
    getFloatLength
}
```
And then import like so:
```js
import * as WebGPUBufferUtils from './WebGPUBufferUtils.js';
```
or
```js
import { getFloatLength } from './WebGPUBufferUtils.js';
```